go-controller/pkg/ovn/address_set_syncer/address_set_sync.go

```diff
@@ -0,0 +1,400 @@
+package address_set_syncer
```

Comment:
nit: a sub-package of `address_set` package might be a better placement (or even in `address_set` package itself). A thought, up to you.

Reply:
yeah I considered this option, but wasn't sure it that makes sense, since `address_set` actually doesn't use it. Also I thought if we need a common syncer for some other db table, we can rename the package to something like `external_ids_syncer` and add it there (and I didn't want to rename is prematurely in case it will always be just address_set).
not a strong opinion though, wdyt?

Reply:
I wouldn't mind a `external_ids_syncer/address_set` package either if that's what we have in mind. But we can move on